cmd/lift/lift.go

```diff
@@ -27,6 +39,24 @@ func lift(chainFile string, inFile string, outFile string) {
 			log.Fatalf("Multiple overlaps???")
 		}
 		i.UpdateLift(interval.LiftIntervalWithChain(overlap[0].(*chain.Chain), i))
+		//special check for lifting over VCF files
+		if faFile != "" {
+			//faFile will be given if we are lifting over VCF data.
+			currVcf = i.(*vcf.Vcf)
+
+			//first question: does the "Ref" match the destination fa at this position.
+			if fasta.QuerySeq(records, currVcf.Chr, int(currVcf.Pos - 1), dna.StringToBases(currVcf.Ref)) {
+				//second question: does the "Alt" also match. Can occur in corner cases such as Ref=A, Alt=AAA. Currently we don't invert but write a verbose log print.
+				if fasta.QuerySeq(records, currVcf.Chr, int(currVcf.Pos - 1), dna.StringToBases(currVcf.Alt)) && Verbose > 0 {
```

Comment:
This looks like it only works with bi-allelic sites.  Is there a check for that somewhere that gives an error to the user?

Reply:
What is the reasoning for restricting this to only bi-allelic sites?